docs/design/ocs-to-ocs-api-server.md

```diff
@@ -0,0 +1,195 @@
+# OCS to OCSs : API Server
+
+## Introduction
+This document defines the API server used for communication between OCS provider and OCS consumer clusters.
+
+
+## Tool
```

Comment:
We already decided on gRPC. 
You can mention that REST can be supported easily.
I don't see the need to mention OpenAPI3

Reply:
removed OpenAPI3